students/Gorthi_kavita/10module/10assignement.py

```diff
@@ -0,0 +1,173 @@
+class donar():
+    list =[]   #this is donar list adding  to the list
+    allist =[]   #to keep track how many times the donar name in the list
+    uniquename = []  # unique donar names
+    listsum = []     # donar names and total amount and number of times donated
+    listadd =[]
+    finalsum = []
```

Comment:
All of these shouldn't be initialized at the top of the class.  Better to have them in an `def __init__` method if they are going to be used through out the class.